test/unit/models/AppealTest.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import java.util.UUID
+
+import uk.gov.hmrc.play.test.UnitSpec
+import utils.Cases._
+
+class AppealTest extends UnitSpec {
+
+  private def randomId(): String = UUID.randomUUID().toString
+
+  "highestAppealFromDecision" should {
+
+    "return None if None is passed" in {
+      //  Given
+      val c = aCase(
+        withoutDecision()
+      )
+
+      // When
+      val res = Appeal.highestAppealFromDecision(c.decision)
+
+      // Then
+      res shouldBe None
+    }
```

Comment:
ScalaTest has a trait [GivenWhenThen](http://doc.scalatest.org/3.0.1-2.12/org/scalatest/GivenWhenThen.html) which you can use if you want BDD style output (see also [FeatureSpec](http://doc.scalatest.org/3.0.1-2.12/org/scalatest/FeatureSpec.html))

Reply:
It was just copy from other class :),  I will keep for now as is